modules/svg/src/SkSVGFilter.cpp

```diff
@@ -25,10 +25,14 @@ bool SkSVGFilter::parseAndSetAttribute(const char* name, const char* value) {
                    "primitiveUnits", name, value));
 }
 
+void SkSVGFilter::applyProperties(SkSVGRenderContext* ctx) const { this->onPrepareToRender(ctx); }
```

Comment:
This is inspired by a similar method in `SkSVGFe`: https://github.com/google/skia/blob/c91e217eb8aebe1497ef7a0032be3cdb2dbc7cbd/modules/svg/src/SkSVGFe.cpp#L91